src/Stratis.Bitcoin.Features.BlockStore/BlockStoreQueue.cs

```diff
@@ -258,7 +258,7 @@ public void AddToPending(BlockPair blockPair)
 
                     batch.Add(item);
 
-                    this.currentBatchSizeBytes += item.Block.GetSerializedSize();
+                    this.currentBatchSizeBytes += item.Block.BlockSize.Value;
```

Comment:
you could potentiall do a 
```
this.currentBatchSizeBytes += tem.Block.BlockSize ?? item.Block.GetSerializedSize()
```

Reply:
It's not possible for a blocksize to be null here- in case it was downloaded it was deserialized. If it was mined- we checked it against the max size rule that serialized it.